app/assets/stylesheets/theme-selector.scss

```
@@ -153,6 +153,35 @@
       font-size: 0.5em;
     }
   }
+
+  @media screen and (max-width: 810px) {
+    .level_tag {
+      display: none;
+    }
+  }
+}
+
+.level-bar__labels {
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
+  padding: 0.3rem var(--su-5);
+  color: black;
+  border-radius: 20px;
```

Just trying to avoid actual units in our CSS:

```suggestion
  border-radius:  var(--su-5); // that stands for 20px
```